### ISSUE TITLE
Upgrade fails

### ISSUE THREAD

**Issue by zefanja** [linuxmuster/linuxmuster-base7#24]
Today I wanted to upgrade `linuxmuster-base7`, but there was an error:

```
linuxmuster-base7 (7.0.1-4ubuntu0) wird eingerichtet ...
Listing '/usr/lib/linuxmuster'...
Compiling '/usr/lib/linuxmuster/constants.py'...
Compiling '/usr/lib/linuxmuster/functions.py'...
Listing '/usr/lib/linuxmuster/setup.d'...
Compiling '/usr/lib/linuxmuster/setup.d/a_ini.py'...
Compiling '/usr/lib/linuxmuster/setup.d/c_general-dialog.py'...
Compiling '/usr/lib/linuxmuster/setup.d/d_templates.py'...
Compiling '/usr/lib/linuxmuster/setup.d/e_fstab.py'...
Compiling '/usr/lib/linuxmuster/setup.d/g_ssl.py'...
Compiling '/usr/lib/linuxmuster/setup.d/h_ssh.py'...
Compiling '/usr/lib/linuxmuster/setup.d/i_samba-provisioning.py'...
Compiling '/usr/lib/linuxmuster/setup.d/j_samba-users.py'...
Compiling '/usr/lib/linuxmuster/setup.d/k_add-server.py'...
Compiling '/usr/lib/linuxmuster/setup.d/m_firewall.py'...
Compiling '/usr/lib/linuxmuster/setup.d/n_linbo.py'...
Compiling '/usr/lib/linuxmuster/setup.d/o_opsi.py'...
Compiling '/usr/lib/linuxmuster/setup.d/p_mail.py'...
Compiling '/usr/lib/linuxmuster/setup.d/w_webui.py'...
Compiling '/usr/lib/linuxmuster/setup.d/z_final.py'...
Creating firewall.fullchain.pem.
cat: /etc/linuxmuster/ssl/firewall.cert.pem: No such file or directory
cat: /etc/linuxmuster/ssl/cacert.pem: No such file or directory
dpkg: Fehler beim Bearbeiten des Paketes linuxmuster-base7 (--configure):
 installed linuxmuster-base7 package post-installation script subprocess returned error exit status 1
Fehler traten auf beim Bearbeiten von:
 linuxmuster-base7
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

Any hints how to fix the certificate problem?

**Comment by zefanja** [comment]
After running `linuxmuster-setup` the upgrade finished.

**Comment by HappyBasher** [comment]
Fixed in 7.0.1-5.
